### ISSUE TITLE
Dir tree display "^G"

### ISSUE THREAD

**Issue by chiplua** [scrooloose/nerdtree#937]
In left dir tree, left of every file name displayed "^G"

### Environment

* Operating System: 
* Vim version `:version`: 
* NERDTree version `git rev-parse --short HEAD`: 
* NERDTree settings applied in your vimrc, if any:
    ```vim
    ```

### Process

1.

**Comment by chiplua** [comment]
- 

- Please visit the link to see the screenshot:http://tieba.baidu.com/p/5996757290

1.

**Comment by PhilRunninger** [comment]
@chiplua, using 'x' may cause problems for you. If your file or directory contains an 'x', NERDTree will not see the whole name. Anything after the 'x' will be dropped. It's better to use a different character. See `:h NERDTreeNodeDelimiter`.

**Comment by ando-takahiro** [comment]
I saw several commits, but the situation was same.  I did not set any settings for nerdtree.  I'm just using simple configuration.

Thanks

**Comment by ando-takahiro** [comment]
I could fix by `brew install vim --with-override-system-vi`.  Sorry for bothering you.

Thanks :)

**Comment by PhilRunninger** [comment]
I'm going to close this issue. Be sure to heed my warning about using 'x' as your delimiter.

**Comment by davidpolverari** [comment]
I still have this same issue even after updating today. Maybe this issue should be reopened?

**Comment by davidpolverari** [comment]
I just tested and up to commit c905a5d7b87051e574c2e72fae57e56c456ba2f4 everything used to work fine. The next commit (ffa8be8) was the one that introduced this issue.

**Comment by bluz71** [comment]
Just updated NERDTree and instantly hit this issue.

I get lots of `^G` symbols as noted in the screenshot above.

My guess is that my font of choice, [Iosevka](https://github.com/be5invis/Iosevka), possibly has a character defined at `\x07`?

It is definitely true that Iosevka has a no glyph defined at `\u00a0` as seen [here](http://typeof.net/Iosevka/specimen.html) in the Latin-1 supplement section.

This fixes the issue for me:

```viml
let g:NERDTreeNodeDelimiter = "\u00a0"
```

I am not sure changing the default node delimiter to the BELL character is an ideal choice.

**Comment by bluz71** [comment]
Actually, it was not the font I was using, it was the because I had set `conceallevel=0` for the NERDTree file type as follows:

```viml
autocmd FileType nerdtree setlocal conceallevel=0
```

Why did I do that?
I suspect I did it many many years ago to fix an issue where [indentLine](https://github.com/Yggdroot/indentLine) was being displaying in the NERDTree window. By turning off conceal, I suspect it disabled indentLine just within NERDTree.

I am not seeing any such weird interactions now between indentLine and NERDTree, once I removed that `autocmd` rule, AND the ugly `^G`s are also gone 😄

It may be worth noting that `g:NERDTreeNodeDelimiter` relies on `conceallevel` and it is best not to fool around with it for NERDTree.

All good now for me.
